src/html/index.html

```diff
@@ -8,6 +8,5 @@
   </head>
   <body>
     <div id="container"></div>
-    <script src="../dist/main.js"></script>

```

Comment:
This isn't needed anymore?

Reply:
This is handled by the webpack html plugin, it will add in the file reference.